code/game/objects/items/storage/toolkit.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/storage/toolkit
+	name = "engineer kit"
+	desc = "An combat engineering toolkit intended to carry electrical and mechanical supplies into combat."
+	icon_state = "toolkit"
+	item_state = "fulton"
+	throw_speed = SPEED_FAST
+	throw_range = 8
+	use_sound = "toolbox"
+	matter = list("plastic" = 2000)
+	can_hold = list(
+		/obj/item/circuitboard,
+		/obj/item/device/flashlight,
+		/obj/item/clothing/glasses/welding,
+		/obj/item/device/analyzer,
+		/obj/item/device/demo_scanner,
+		/obj/item/device/reagent_scanner,
+		/obj/item/device/t_scanner,
+		/obj/item/stack/cable_coil,
+		/obj/item/cell,
+		/obj/item/device/assembly,
+		/obj/item/stock_parts,
+		/obj/item/explosive/plastic,
```

Comment:
Breaching charges? (They might be a c4 subtype already, i don't remember)

Reply:
I think that Breaching Charges are their own path off of explosives so they should probably be added here

Reply:
They're a subtype